modules/ols-enabling-mcp-server.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+// * lightspeed-docs-main/configure/ols-configuring-openshift-lightspeed.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ols-enabling-mcp-server_{context}"]
+= Enabling MCP server
+
+Enable the Model Context Protocol (MCP) server so that a large language model (LLM) can securely access an external tool, such as a database or API, and obtain real-time updates.
+
+.Prerequisites
+
+* You have installed the the {ols-long} Operator.
+
+* You have configured a large language model provider.
+
+* You have deployed the {ols-long} service.
+
+.Procedure
+
+. Open the {ols-long} `OLSconfig` custom resource (CR) file by running the following command:
++
+[source,terminal]
+----
+$ oc edit olsconfig cluster
+----
+
+. Modify the `spec.ols.featureGates` specification to include MCP server information.
```

Comment:
add `MCPServer` to the list `spec.ols.featureGates` to activate MCP Server feature.
Then define connection details for each MCP server in the list `mcpServers`.